### PR TITLE
[12.0][ADD] fleet_vehicle_inspection_template

### PR DESCRIPTION
**Pull request by marcelsavegnago**
cc @brian10048 @mamcode

### CODE REVIEW

**Comment by marcelsavegnago** [comment]
cc @eduaparicio

**Comment by marcelsavegnago** [comment]
@OCA/fleet-maintainers ready to review

File: fleet_vehicle_inspection_template/i18n/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 2.4.1\n"
+
+#. module: fleet_vehicle_inspection_template
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection_template.field_fleet_vehicle_inspection_template__message_needaction
+msgid "Action Needed"
+msgstr "Acão Nesessária"
```
Comment (eduaparicio) [inline fleet_vehicle_inspection_template/i18n/pt_BR.po]: Ação Necessária

File: fleet_vehicle_inspection_template/i18n/pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+#. module: fleet_vehicle_inspection_template
+#: model:ir.model,name:fleet_vehicle_inspection_template.model_fleet_vehicle_inspection_template
+msgid "Fleet Vehicle Inspection Template"
+msgstr "Modelo de inspeção de veículos da frota"
+
+#. module: fleet_vehicle_inspection_template
+#: model:ir.model,name:fleet_vehicle_inspection_template.model_fleet_vehicle_inspection_template_line
+msgid "Fleet Vehicle Inspection Template Line"
+msgstr "Linha de Modelo de Inspeção de Veículos da Drota"
```
Comment (eduaparicio) [inline fleet_vehicle_inspection_template/i18n/pt_BR.po]: "Linha de Modelo de Inspeção de Veículos da Frota"

File: fleet_vehicle_inspection_template/i18n/pt_BR.po [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+
+#. module: fleet_vehicle_inspection_template
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection_template.field_fleet_vehicle_inspection_template__activity_date_deadline
+msgid "Next Activity Deadline"
+msgstr "Prazo final para Próxima Atividade"
+
+#. module: fleet_vehicle_inspection_template
+#: model:ir.model.fields,field_description:fleet_vehicle_inspection_template.field_fleet_vehicle_inspection_template__activity_summary
+msgid "Next Activity Summary"
+msgstr "Próximo Sumário de Atividade"
```
Comment (eduaparicio) [inline fleet_vehicle_inspection_template/i18n/pt_BR.po]: Sugiro algo como "Resumo da Próxima Atividade"

**Review by eduaparicio — APPROVED** [review]
I'm leaving minor suggestions on portuguese translation, but all the important code looks good.
Congrats, @marcelsavegnago !!

File: fleet_vehicle_inspection_template/i18n/pt_BR.po (hunk 1 above)
Comment (marcelsavegnago) [reply fleet_vehicle_inspection_template/i18n/pt_BR.po]: Done

File: fleet_vehicle_inspection_template/i18n/pt_BR.po (hunk 2 above)
Comment (marcelsavegnago) [reply fleet_vehicle_inspection_template/i18n/pt_BR.po]: Done

**Comment by marcelsavegnago** [comment]
@rvalyi @renatonlima @mileo If you have time, take a look at this PR please.

**Comment by marcelsavegnago** [comment]
ping @brian10048 @mamcode

File: fleet_vehicle_inspection_template/views/fleet_vehicle_inspection_template.xml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="fleet_vehicle_inspection_template_act_window">
+        <field name="name">Inspection Template</field>
+        <field name="res_model">fleet.vehicle.inspection.template</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[]</field>
+        <field name="context">{}</field>
```
Comment (mamcode) [inline fleet_vehicle_inspection_template/views/fleet_vehicle_inspection_template.xml]:
```suggestion
```

Comment (mamcode) [reply fleet_vehicle_inspection_template/views/fleet_vehicle_inspection_template.xml]: These lines I think are not necessary, they can be eliminated but as you prefer it is fine, it is not blocking :)

File: fleet_vehicle_inspection_template/views/fleet_vehicle_inspection_template_line.xml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="fleet_vehicle_inspection_template_line_act_window">
+        <field name="name">Inspection Template Line</field>
+        <field name="res_model">fleet.vehicle.inspection.template.line</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[]</field>
+        <field name="context">{}</field>
```
Comment (mamcode) [inline fleet_vehicle_inspection_template/views/fleet_vehicle_inspection_template_line.xml]: The same here.
```suggestion
```

**Review by mamcode — APPROVED** [review]
LGTM, thanks :+1:

**Comment by ivantodorovich** [comment]
LGTM :+1: , can you squash your commits though?

**Comment by marcelsavegnago** [comment]
> LGTM 👍 , can you squash your commits though?

@ivantodorovich Thanks. Squash its done.

**Comment by ivantodorovich** [comment]
Thanks!

/ocabot merge nobump

**Pull request merged**